clean/baton_rouge_csd_pprr.py

```diff
@@ -1,3 +1,4 @@
+from clean.scott_pd_pprr import clean_rank
```

Comment:
This form of import is bad. What will happen if you want to update Scott PD? If you are not careful anything that import from `clean.scott_pd_pprr` will break and you have to fix them. As a general rule, things that can be shared must live under `lib` package. If you think this function is generic enough that it can be reused then please put it under `lib` package.

Reply:
DIdn't mean to do this. I've noticed that visual studio has been automatically calling functions that share the same name from different scripts. I missed this creeping into baton_rouge_csd_pprr.py and kenner_pd.py. Will resolved now.